tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -40,6 +40,30 @@ TEST_F(NVFuserTest, TestIsSharded) {
   EXPECT_ANY_THROW(isSharded(c));
 }
 
+TEST_F(NVFuserTest, TestPropagateSharding) {
```

Comment:
```suggestion
TEST_F(ShardingTest, PropagateSharding) {
```
and at the beginning of the file say `using ShardingTest = NVFuserTest`. I'd do this for other NVFuserTests in this file as well. This gives the tests a less overloaded fixture name for easy filtering. For example, `bin/nvfuser_tests --gtest_filter=ShardingTest*`.

Reply:
That's nice way to organize our tests better. Made it MultiDeviceUtilsTest and bundled it with others :D